VacationPlanner-ConsoleApp/src/main/java/com/isa/Team.java

```diff
@@ -0,0 +1,7 @@
+package com.isa;
+
+public class Team {
+    private String teamName;
+    private Employee[] employees = new Employee[5];
```

Comment:
**notatka:**
Rozwiązanie z tablicą jest tymczasowe, ponieważ nie było jeszcze na warsztatach kolekcji (np. List lub Set).
Później możemy to zmienić na wygodniejszy w 'obsłudze' zestaw danych 😉